CHANGES.txt

```diff
@@ -1,3 +1,8 @@
+
+
+Added:
```

Comment:
nit: The changelog update can also wait until the release PR.  Not a blocker (more of an FYI) because this PR is branched off dev version [2.5dev](https://github.com/planetlabs/planet-client-python/blob/fb-support-1026/planet/__version__.py#L1).  Just remember to add the release version to this changelog entry in the release PR!

Reply:
Cool. I tried to look a bit to see what the best practice was - to put it in now or wait till release. Feel free to remove if the standard is just to have whoever is releasing write the notes - just figured I'd try to be helpful.